AdobeStockAdminUi/Test/Mftf/ActionGroup/AdminAdobeStockSetConfigActionGroup.xml

```diff
@@ -11,8 +11,8 @@
     <actionGroup name="AdminAdobeStockSetConfigActionGroup">
         <arguments>
             <argument name="enabled" type="string" defaultValue="{{AdobeStockConfigDataEnabled.value}}"/>
-            <argument name="apiKey" type="string" defaultValue="{{AdobeStockConfigDataApiKey.value}}"/>
```

Comment:
May you please explain the reason of removing the default values here? There can be cases, when we need to configure Adobe Stock with some wrong credentials, check the following test: https://github.com/magento/adobe-stock-integration/blob/develop/AdobeStockAdminUi/Test/Mftf/Test/AdminCannotAccessStockImagesWithWrongCredentialsTest.xml#L24

Reply:
So I copied this ActionGroup into a separate one called `AdminAdobeStockSetCorrectConfigActionGroup.xml` which hard-codes the correct credentials in, while this one is kept for the tests that e.g. enter incorrect credentials and test those cases out. I removed the default values here as, according to the [MFTF documentation, using Credentials only works with free MFTF elements](https://devdocs.magento.com/mftf/docs/credentials.html#use-credentials-in-a-test): `fillField`, `magentoCLI`, and `createData`. If it's possible to use credentials in arguments, then we can revert my changes here and consolidate on only using this one single action and arguments with credentials, but I didn't try because of what I read in the documentation.